### ISSUE TITLE
[NEW BUILD - DEFAULT / 19w31b] Hotfixes, better shield

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Skywanderers#1406]
Mainly hotfixes, plus I did a cleanup / optimization of the shield.
I also added a couple of frosted glass windows as it was possible thanks to some shield rendering workflow optimizations (and also Blurred UI, it looks great with the minimalist style).

I am now full on the damage / armor model, that will be next update!
Thanks!

**Comment by ProPeach** [comment]
Not sure the frosted glass is working as intended. In the attached video you can see the plain frosted glass with the blue marker and the patterned frosted glass with the red marker.
[Skywanderers frosted glass bug.zip](https://github.com/tsunamayo/Skywanderers/files/3464874/Skywanderers.frosted.glass.bug.zip)
It looks like the plain glass has a red arrow sticker on it and the patterned glass has a red square.

Also, the drag placement of the frosted glass bricks feels strange, it doesn't act like regular bricks when dragged in the same direction. I'd suggest making the dragging identical to the regular brick to maintain continuity across the system.

The shield effect is really nice though! It looks like ice crystals

**Comment by Majikmonster** [comment]
About the frosted glass: It's not translucent like the preview on Twitter or like the video Peach posted for me 😕 
![20190804095750_1](https://user-images.githubusercontent.com/51390907/62424615-03b8aa80-b69f-11e9-94cb-2b55b3cbc958.jpg)
Also, the decals are disordered in the codex and have reflected that change on all the ones applied to ships (the carpet looks a bit odd now :P)
![20190804092014_1](https://user-images.githubusercontent.com/51390907/62424639-74f85d80-b69f-11e9-9417-4e463d57284e.jpg)

**Comment by ProPeach** [comment]
I noticed that you have to paint them for them to actually turn clear otherwise they look like the image Maj posted, sorry I should have made that clear

**Comment by tsunamayo** [comment]
Yes sorry I must have messed up something during the build upload, I keep you posted.
